src/control/system/pool.go

```diff
@@ -24,6 +24,10 @@ const (
 	PoolServiceStateReady
 	// PoolServiceStateDestroying indicates that the pool service is being destroyed
 	PoolServiceStateDestroying
+	// PoolServiceStateDegraded indicates that the pool service is being Degraded
```

Comment:
Should probably offer a bit more description about what "degraded" means here, also "being degraded" doesn't make sense in this context.